File: src/test/java/cernx/colt/matrix/tlong/impl/SparseCCMLongMatrix2DTest.java
package cernx.colt.matrix.tlong.impl;

import cernx.colt.matrix.tlong.LongMatrix2DTest;

public class SparseCCMLongMatrix2DTest extends LongMatrix2DTest {

    public SparseCCMLongMatrix2DTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCMLongMatrix2D(NROWS, NCOLUMNS);
        B = new SparseCCMLongMatrix2D(NROWS, NCOLUMNS);
        Bt = new SparseCCMLongMatrix2D(NCOLUMNS, NROWS);
    }

}

File: code/src/test/scala/cern/colt/matrix/tlong/impl/SparseCCMLongMatrix2DTest.scala
package cern.colt.matrix.tlong.impl

import cern.colt.matrix.tlong.LongMatrix2DTest
//remove if not needed
import scala.collection.JavaConversions._

class SparseCCMLongMatrix2DTest(arg0: String) extends LongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseCCMLongMatrix2D(NROWS, NCOLUMNS)
    B = new SparseCCMLongMatrix2D(NROWS, NCOLUMNS)
    Bt = new SparseCCMLongMatrix2D(NCOLUMNS, NROWS)
  }
}
